pkg/util/k8sutil/k8sutil.go

```diff
@@ -220,7 +222,38 @@ func addOwnerRefToObject(o metav1.Object, r metav1.OwnerReference) {
 	o.SetOwnerReferences(append(o.GetOwnerReferences(), r))
 }
 
-func NewEtcdPod(m *etcdutil.Member, initialCluster []string, clusterName, state, token string, cs spec.ClusterSpec, owner metav1.OwnerReference) *v1.Pod {
+func NewPVC(m *etcdutil.Member, cs spec.ClusterSpec, clusterName, namespace string, pvProvisioner string, owner metav1.OwnerReference) *v1.PersistentVolumeClaim {
+	name := m.PVCName()
+	storageClassName := storageClassPrefix + "-" + path.Base(pvProvisioner)
+	pvc := &v1.PersistentVolumeClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels: map[string]string{
+				"etcd_node":    m.Name,
+				"etcd_cluster": clusterName,
+				"app":          "etcd",
+			},
+		},
+		Spec: v1.PersistentVolumeClaimSpec{
+			StorageClassName: &storageClassName,
```

Comment:
Here I'm using Spec.StorageClassName instead of the old annotation `volume.beta.kubernetes.io/storage-class` like it's used in the backup pvc: https://github.com/coreos/etcd-operator/blob/master/pkg/util/k8sutil/backup.go#L74 . Should this be updated?